OracleHTTPServer/dockerfiles/12.2.1.4.0/container-scripts/provisionOHS.sh

```diff
@@ -17,7 +16,7 @@
 function _term() {
    echo "Stopping container."
    echo "SIGTERM received, shutting down the server!"
-   ${WLST_HOME}/wlst.sh /u01/oracle/container-scripts/stop-ohs.py
+   ${WLST_HOME}/wlst.sh /u01/oracle/stop-ohs.py
```

Comment:
You are copying all necessary scripts under /u01/oracle/container-scripts in Dockerfile. So, I think the old line was correct.

Reply:
I will change the Dockerfile to place the scripts under /u01/oracle otherwise this script will not work with Image Tool